deployment/ccip/changeset/cs_chain_contracts.go

```diff
@@ -1143,4 +1143,8 @@ func DefaultFeeQuoterDestChainConfig() fee_quoter.FeeQuoterDestChainConfig {
 		NetworkFeeUSDCents:                1,
 		ChainFamilySelector:               [4]byte(evmFamilySelector),
 	}
+	if !configEnabled {
+		feeQuoterDestChainConfig.IsEnabled = false
+	}
+	return feeQuoterDestChainConfig
```

Comment:
Change this directly no need for condition check
```suggestion
	
	return feeQuoterDestChainConfig
```